### PR TITLE
Prep javascript variables | "completed"

### PR DESCRIPTION
**Pull request by normansilvera**

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+
+    var likesVegetables = true;
+    console.log(likesVegetables);
+
+    var nothing = null;
+    console.log(nothing);
+
+    var never = "";
```
Comment (thebearingedge) [inline prep-javascript-variables/index.html]: You can leave off the `= ""`

```js
var never;
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by normansilvera** [comment]
Screen Shot 2021-06-15 at 3 16 35 PM

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (normansilvera) [reply prep-javascript-variables/index.html]: Screen Shot 2021-06-15 at 3 16 35 PM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
careful, this JavaScript doesn't parse now. you have to leave out the `=` as well as in my previous example

```js
var never;
```

**Comment by normansilvera** [comment]
Screen Shot 2021-06-15 at 3 19 32 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**